scripts/csops/csi

```diff
@@ -219,11 +218,6 @@ if [[ $rebuild_services_needed == true ]]; then
 
 	echo "Deploying services..."
 
-	if ! $CSDOWN; then
```

Comment:
Let's replace the CSDOWN with a csstatus check.  We don't want to be running the "ant undeploy deploy" command when CollectionSpace is running.

Reply:
csstatus runs at the beginning and the script will fail if cspace is running.

At your request, I did remove the force option.